app/services/idv/vendor_validator.rb

```diff
@@ -25,5 +25,22 @@ def idv_agent
         vendor: (idv_session.vendor || idv_vendor.pick)
       )
     end
+
+    def result
+      @_result ||= try_submit
+    end
+
+    def try_agent_action
+      yield
+    rescue => err
+      err_msg = err.to_s
```

Comment:
Yeah we're using NewRelic as our exception thingy, so the syntax is `NewRelic::Agent.notice_error`

The application logs stuff is going to be manual, NewRelic already alerts when it sees an uptick in exceptions per transaction (transaction = web request or background job), but since this is getting eaten, NewRelic wouldn't know